zilencer/views.py

```diff
@@ -44,6 +45,7 @@
     RemoteRealmServerMismatchError,
     RemoteServerDeactivatedError,
 )
+from zerver.lib.outgoing_http import OutgoingSession
```

Comment:
OutgoingSession exists to be subclassed, not used directly.  The goal is that in the future we will lock down (via smokescreen config) what hosts various different OutgoingSession types can talk to, as hardening.